### ISSUE TITLE
Need help with getting prob/logits vectors and inversion

### ISSUE THREAD

**Issue by LuoXiaoxi-cxq** [vec2text/vec2text#87]
Hi, thank you for your awesome code!

However, when I tried to reproduce your results in _Language Model Inversion_, I couldn't find a quick start guide. Most examples and existing issues are about embedding inversion. The example of how to evaluate on the Python-Alpaca dataset in the README is nice, but it's a little "over-wrapped" for me—I just want to get the prob/logits vectors first, maybe modify the vectors, and then inverse them.

Would you please help me? Thank you!

**Comment by LuoXiaoxi-cxq** [comment]
By the way, why do we need the corrector model for *Language Model Inversion*? In test_llama_inv_vec2text.py, corrector model `jxm/t5-base___llama-7b___one-million-instructions__correct` is loaded, but I did not find relavant information in your paper. Maybe I have lost something. Thanks!
